xbmc/cores/playercorefactory/PlayerCoreFactory.cpp

```diff
@@ -14,6 +14,7 @@
 #include "URL.h"
 #include "cores/IPlayerCallback.h"
 #include "cores/paplayer/PAPlayer.h"
+#include "cores/VideoPlayer/Interface/Addon/InputStreamConstants.h"
```

Comment:
> Included header files have to be sorted (case sensitive) alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers